src/styles/blocks/card.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-card;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
```

Comment:
The `margin` shorthand property should list values in the order: top, right, bottom, left. Since the left and right margins are the same, you can use the shorthand `margin: 0 auto;` without specifying the bottom margin separately, and then add `margin-bottom: 40px;` on a new line for clarity.